pkgs/development/libraries/mesa/default.nix

```diff
@@ -2,14 +2,15 @@
 , pkgconfig, intltool, autoreconfHook
 , file, expat, libdrm, xorg, wayland, wayland-protocols, openssl
 , llvmPackages, libffi, libomxil-bellagio, libva-minimal
-, libelf, libvdpau, valgrind-light, python2, python2Packages
+, libelf, libvdpau, python2, python2Packages
 , libglvnd
 , enableRadv ? true
 , galliumDrivers ? null
 , driDrivers ? null
 , vulkanDrivers ? null
 , eglPlatforms ? [ "x11" ] ++ lib.optionals stdenv.isLinux [ "wayland" "drm" ]
 , OpenGL, Xplugin
+, withValgrind ? stdenv.hostPlatform.isLinux && !stdenv.hostPlatform.isAarch32, valgrind-light
```

Comment:
In which cases do we default the package to null when it's possible to disable via an option?
You didn't but the `enableRadv` option does.

Reply:
I prefer to use ```with*``` flags as they are more obvious. I guess we can always do something like ```withValgrind ? valgrind != null```